projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -1005,11 +1005,11 @@ function maybe_close_comments( $default_comments_open, $post_id ) {
 }
 
 /**
- * Gate access to existing comments
+ * Gate access to existing comments.
  *
- * @param string $comment The comment.
+ * @param \WP_Comment $comment The comment.
  *
- * @return string
+ * @return \WP_Comment|null
```

Comment:
Maybe you could update that check below so it ensures `$comment` is a `WP_Comment`, instead of just `empty`?

`if ( ! $comment instanceof \WP_Comment ) {`